pkg/common/point.go

```diff
@@ -0,0 +1,17 @@
+package common
+
+// Point represent a data point in a series
+// ie: cpu.usage <2016-11-12-12:41:33, 0.062>, <2016-11-12-12:41:34, 0.078>
+// cpu.usage is a series and it has two points, which shows different usage at different time
+
+// IntPoint has integer value and unix nano timestamp in int64
+type IntPoint struct {
+	V int
+	T int64
```

Comment:
How do you think about using empty interface `interface{}` for V?
https://tour.golang.org/methods/14
How do you think about using `time.Time` for T?
Also, T could be confusing since T refers to "type" in many docs.